pkg/reconciler/service/workerpool.go

```diff
@@ -23,12 +16,11 @@ type workPoolBuilder struct {
 }
 
 type WorkerPool struct {
-	debug                bool
-	logger               *zap.SugaredLogger
-	antsPool             *ants.Pool
-	PoolID               string
-	occupancyCallbackURL string
-	newRunnerFct         func(context.Context, *reconciler.Task, callback.Handler, *zap.SugaredLogger) func() error
+	debug        bool
+	logger       *zap.SugaredLogger
+	antsPool     *ants.Pool
+	CallbackURL  string
```

Comment:
This is actually nothing the worker pool should have to know as it's occupancy-tracker specific.

Can'T we move it as field to the occupany-tracker struct?